data-processing/tests/test_extract_definitions.py

```diff
@@ -23,10 +23,11 @@ def model():
 @pytest.mark.slow
 def test_model_extracts_simple_definitions(model: DefinitionDetectionModel):
     prediction_type = "W00"
-    features = model.featurize("Neural networks are machine learning models.")
-    intents, slots, _ = model.predict_batch([features], prediction_type)
+    features = model.featurize("Neural networks are a type of machine learning models.")
```

Comment:
What is the reason for the change to this test?

Reply:
The test was failing here. I inspected the logits and could not understand why. I think this was a borderline case, in the earlier model. The text based part of the model is still not as good as the other two heads (but it is actually better than the previous version of the model in aggregate, just not this case surprisingly). So, I rephrased it a little, and it worked. 

But I think we need to somewhere record model test cases (one could argue that's what the dev set is, but I'm not sure where).

Reply:
Then let us include the original test case as a skipped test case to document that a future definition detector should probably extract it, as is already done for this test case in the same file https://github.com/allenai/scholarphi/blob/8632c464f58c0b787ba2dd363b5be449f5b1fa98/data-processing/tests/test_extract_definitions.py#L470-L479

Reply:
Done!